sdk/security/keyvault/azadmin/ci.yml

```diff
@@ -26,16 +26,15 @@ extends:
   parameters:
     TimeoutInMinutes: 120
     ServiceDirectory: 'security/keyvault/azadmin'
-    RunLiveTests: true
     UsePipelineProxy: false
-    MatrixConfigs:
-      - Name: keyvault_test_matrix_addons
-        Path: sdk/security/keyvault/azadmin/platform-matrix.json
-        Selection: sparse
-        GenerateVMJobs: true
 
-    ${{ if not(contains(variables['Build.DefinitionName'], 'tests-weekly')) }}:
-        # Due to the high cost of Managed HSMs, which keyvault-admin requires, we only want to run
-        # the live tests weekly.
-        MatrixFilters:
-          - ArmTemplateParameters=^(?!.*enableHsm.*true)
+    # Due to the high cost of Managed HSMs, which keyvault-admin requires, we only want to run
+    # the live tests weekly or for releases (manual builds).
+    ${{ if or(contains(variables['Build.DefinitionName'], 'weekly'), eq(variables['Build.Reason'], 'Manual')) }}:
```

Comment:
🤔 I'm uncertain whether this would be better, but you could check only the build name. Then when you want to run these tests manually, you'd run `go - azadmin - weekly`

Reply:
Was talking to Ben about this. Set it that way so that the live tests will run when the go - azadmin pipeline is triggered manually for releases. Don't want the release to go out without making sure the live tests work.